packages/core/src/physics/joint/Joint.ts

```
@@ -24,6 +24,8 @@ export abstract class Joint extends Component {
   private _force = Infinity;
   private _torque = Infinity;
   private _automaticConnectedAnchor = true;
+  @ignoreClone
+  private _updateConnectedActualAnchor: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)